docs/serving/creating-domain-mappings.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Creating a Mapping between a Custom Domain Name and a Knative Service (Alpha)"
+linkTitle: "Creating Domain Mappings (Alpha)"
+weight: 64
+type: "docs"
+---
+
+If the Domain Mapping feature is installed, you can serve
```

Comment:
This sentence sounds like the user needs to install an optional components when installing knative..
Is this true or you mean that user use a knative 0.19+ that contains this CRD and that the state of the feature is alpha and we want feedback early

Reply:
Yes, the user does need to install an additional component -- it's not installed by default in this release (there's an extra yaml file, much like installing the monitoring bundle or HPA autoscaler). I'll PR an update to install docs separately with that optional step.

Reply:
Add a reference and link here you don’t have to put the actual steps here

Reply:
I will when I've added that step to the docs :)

Reply:
I think we need to add to this doc the limitations, that not all net ingress support this feature, you can list them here, or mention in the install doc and point there from here.